packages/event-bus-local/src/services/event-bus-local.ts

```diff
@@ -0,0 +1,76 @@
+import { Logger, MedusaContainer } from "@medusajs/medusa"
+import { EventBusTypes } from "@medusajs/types"
+import { EventBusUtils } from "@medusajs/utils"
+import { EventEmitter } from "events"
+import { EntityManager } from "typeorm"
+
+type InjectedDependencies = {
+  logger: Logger
+}
+const eventEmitter = new EventEmitter()
+
+export default class LocalEventBusService extends EventBusUtils.AbstractEventBusModuleService {
+  protected readonly logger_: Logger
+  protected readonly manager_: EntityManager
```

Comment:
Do we need this `manager_`?

Reply:
in reality they are already in the base class and accessible, so we don't have to have them 👍